### DISCUSSION TITLE
MySQLConnector error Unsupported command(COM_RESET_CONNECTION)

### DISCUSSION THREAD

**Discussion by Rudy-ppt** [apache/doris#36136] [A - General / Q&A]
Hi,

using MySQLConnector  in C#, the first connection open works, the second returns the error "Unsupported command(COM_RESET_CONNECTION)". Maybe .net driver reuses the TCP connection and it detects 5.7 as the MySQL version.
(MySqlConnector will use COM_RESET_CONNECTON (higher server version) or COM_CHANGE_USER (lower server version) to reset state of connection.)

Example:

MySqlConnection conn = new MySqlConnection(cstring);
conn.Open();
conn.Close();
conn.Open();  //ERROR
conn.Close();

See a similar case here https://github.com/apache/shardingsphere/issues/6861

Is it possible to fix it?

Thanks,
Rudy

**Comment by Rudy-ppt** [comment]
I found a workaround, specifying the ConnectionReset=False option, however this results the connection state is not reset, meaning that session variables and other session state changes from any previous use of the connection are carried over.
I don't know if on Doris there may be problems reusing the same connection, otherwise it is necessary to completely disable the connection pool.
Has anyone had any problems with the connection pool?

**Comment by echo-hhj** [comment]
Currently not supported, it is expected to be available in the next quarter for the COM_RESET_CONNECTION command.
